src/Models/Node.ts

```diff
@@ -0,0 +1,31 @@
+import { Model, InitOptions, ModelAttributes, DataTypes, Sequelize } from "sequelize";
+
+export class Node extends Model {
+    // to do add ID, updatedAt, createdAt
```

Comment:
Resolve or remove those todos. TODOs are fine in development but can be merged only if something bigger should be made but this can be easily done and it is part of the task.